src/arithmetic/x86_64_mont.rs

```diff
@@ -157,8 +163,13 @@ pub(super) unsafe fn mul_mont_gather5_amm(
     let n = n.as_flattened();
     // SAFETY: We cannot assert that `power` is in range because it is secret.
     // TODO: Create a `Window5` type that is guaranteed to be in range.
+    let mul_mont_gather5 = if maybe_adx_bmi1_bmi2.is_some() {
+        bn_mulx4x_mont_gather5
+    } else {
+        bn_mul4x_mont_gather5
+    };
```

Comment:
This Codecov warning is stale. Codecov shows it is covered and also I verified locally.